### PR TITLE
[Localization] Added Korean localization for challenges 

### PR DESCRIPTION
**Pull request by sodaMelon**
## What are the changes?

Korean Localization for the challenges
## Why am I doing these changes?

Because they weren't localized

## What did change?

I translated all the keys into Korean
### Screenshots/Videos

Don't have any, but I'm 100% sure the edited texts fits

## How to test the changes?

Set your game in Korean and start a new Challange

## Checklist
- [x] There is no overlap with another PR?
- [x] The PR is self-contained and cannot be split into smaller PRs?
- [x] Have I provided a clear explanation of the changes?
- [x] Have I tested the changes (manually)?
    - [x] Are all unit tests still passing? (`npm run test`)
- [x] Are the changes visual?
  - [ ] Have I provided screenshots/videos of the changes?

### CODE REVIEW

**Comment by sodaMelon** [comment]
@returntoice hi 
What you were talking about in Discord, I finished.

File: src/locales/ko/challenges.ts [hunk 1]
```diff
@@ -1,67 +1,67 @@
...
+  "singleType.value.4": "독",
+  "singleType.desc.4": "독 타입의 포켓몬만 사용할 수 있습니다.",
+  "singleType.value.5": "땅",
+  "singleType.desc.5": "땅 타입의 포켓몬만 사용할 수 있습니다.",
+  "singleType.value.6": "바위 ",
+  "singleType.desc.6": "바위 타입의 포켓몬만 사용할 수 있습니다.",
+  "singleType.value.7": "벌레",
+  "singleType.desc.7": "벌레 타입의 포켓몬만 사용할 수 있습니다.",
+  "singleType.value.8": "고스트",
+  "singleType.desc.8": "고스트 타입의 포켓몬만 사용할 수 있습니다G.",
```
Comment (GINK-SS) [inline src/locales/ko/challenges.ts:46]:
```suggestion
  "singleType.desc.8": "고스트 타입의 포켓몬만 사용할 수 있습니다.",
```

오타 검수

**Review by GINK-SS — COMMENTED** [review]
추가로 설정 안함의 desc에서 '당신은' 부분은 빼도 괜찮을 것 같습니다.

**Review by Ohmry — APPROVED** [review]
LGTM! 👍

File: src/locales/ko/challenges.ts (hunk 1 above)
Comment (sodaMelon) [reply src/locales/ko/challenges.ts:46]: thanks !!!!!!

**Review by returntoice — APPROVED** [review]
Good!

**Comment by sodaMelon** [comment]
@GINK-SS  Thanks to you, I fixed the sentence clearly. And it got shorter!

**Review by Tempo-anon — APPROVED** [review]

**Pull request merged**